src/features/pages/consultancy-page.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Consultancy Page</title>
+</head>
+<body>
+    <div>Consultancy Page</div>
+    
+</body>
+</html>
```

Comment:
Your PR links issue 12, but you have here changes for another issue. Always make sure you work on one issue at a time.